src/app/clusters/scenes-server/scenes-server.cpp

```diff
@@ -0,0 +1,940 @@
+                response.status = static_cast<uint8_t>(Protocols::InteractionModel::Status::Failure);
+                ctx.mCommandHandler.AddResponse(mPath, response);
+                return;
+            }
+
+            scene.mStorageId = SceneStorageId(mPath.mEndpointId, SceneId, req.groupIdentifierTo);
+
+            err = mSceneTable->SetSceneTableEntry(ctx.mCommandHandler.GetAccessingFabricIndex(), scene);
+            if (err != CHIP_NO_ERROR)
+            {
+                if (err == CHIP_ERROR_NO_MEMORY)
+                {
+                    response.status = static_cast<uint8_t>(Protocols::InteractionModel::Status::ResourceExhausted);
+                    ctx.mCommandHandler.AddResponse(mPath, response);
+                }
+                else
+                {
+                    response.status = static_cast<uint8_t>(Protocols::InteractionModel::Status::Failure);
+                    ctx.mCommandHandler.AddResponse(mPath, response);
+                }
+                return;
+            }
+        }
+
+        response.status = static_cast<uint8_t>(Protocols::InteractionModel::Status::Success);
+        ctx.mCommandHandler.AddResponse(mPath, response);
+        return;
+    }
+
+    SceneTableEntry scene(SceneStorageId(mPath.mEndpointId, req.sceneIdentifierFrom, req.groupIdentifierFrom));
+    err = mSceneTable->GetSceneTableEntry(ctx.mCommandHandler.GetAccessingFabricIndex(), scene.mStorageId, scene);
+    if (err != CHIP_NO_ERROR)
+    {
+        if (err == CHIP_ERROR_NOT_FOUND)
+        {
+            response.status = static_cast<uint8_t>(Protocols::InteractionModel::Status::NotFound);
+            ctx.mCommandHandler.AddResponse(mPath, response);
+        }
+        else
+        {
+            response.status = static_cast<uint8_t>(Protocols::InteractionModel::Status::Failure);
+            ctx.mCommandHandler.AddResponse(mPath, response);
+        }
+        return;
+    }
+
+    scene.mStorageId = SceneStorageId(mPath.mEndpointId, req.sceneIdentifierTo, req.groupIdentifierTo);
+
+    err = mSceneTable->SetSceneTableEntry(ctx.mCommandHandler.GetAccessingFabricIndex(), scene);
+    if (err != CHIP_NO_ERROR)
+    {
+        if (err == CHIP_ERROR_NO_MEMORY)
+        {
+            response.status = static_cast<uint8_t>(Protocols::InteractionModel::Status::ResourceExhausted);
+            ctx.mCommandHandler.AddResponse(mPath, response);
+        }
+        else
+        {
+            response.status = static_cast<uint8_t>(Protocols::InteractionModel::Status::Failure);
+            ctx.mCommandHandler.AddResponse(mPath, response);
+        }
+        return;
+    }
+
+    response.status = static_cast<uint8_t>(Protocols::InteractionModel::Status::Success);
+    ctx.mCommandHandler.AddResponse(mPath, response);
+}
+
+} // namespace Scenes
+} // namespace Clusters
+} // namespace app
+} // namespace chip
+
+// EMBER AF CALLBACKS:
+// Those have been added to avoid linker issue with zap and should never get called
```

Comment:
Instead of doing this hack, please add the cluster to the list at https://github.com/project-chip/connectedhomeip/blob/a3b400568cbace2b8fd328325d93685fc2ad5512/src/app/common/templates/config-data.yaml#L26